packages/web-twig/src/Resources/components/ScrollView/stories/ScrollViewScrollingOverflowDecorators.twig

```diff
@@ -1,13 +1,13 @@
 <section class="docs-Section">
 
-    <h2 class="docs-Heading">Scrolling Indicators</h2>
+    <h2 class="docs-Heading">Scrolling Overflow Decorators</h2>
```

Comment:
```suggestion
    <h2 class="docs-Heading">Overflow Decorators</h2>
```

Please shorten it also in the other files. ⬇️

Reply:
Fixed.